sonar-vbnet-plugin/pom.xml

```diff
@@ -227,80 +228,61 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-antrun-plugin</artifactId>
+        <version>3.1.0</version>
+        <executions>
+          <execution>
+            <id>copy-analyzer-data</id>
+            <phase>validate</phase>
+            <configuration>
+              <exportAntProperties>true</exportAntProperties>
+              <target>
+                <!-- ITs requires the SonarAnalyzer-VERSION.zip to be present so don't remove these lines -->
+                <copy todir="${sonarAnalyzer.workDirectory}/SonarAnalyzer.VisualBasic">
+                  <fileset dir="${analyzers.directory}/packaging/binaries/">
+                    <include name="Google.Protobuf.dll"/>
+                    <include name="SonarAnalyzer.dll"/>
+                    <include name="SonarAnalyzer.CFG.dll"/>
+                    <include name="SonarAnalyzer.VisualBasic.dll"/>
+                  </fileset>
+                  <fileset dir="${project.build.directory}/../..">
+                    <include name="THIRD-PARTY-NOTICES.txt"/>
+                  </fileset>
+                </copy>
+                <zip destfile="${sonarAnalyzer.workDirectory}/static/SonarAnalyzer-${project.version}.zip"
+                     basedir="${sonarAnalyzer.workDirectory}/SonarAnalyzer.VisualBasic"/>
+                <copy todir="${sonarAnalyzer.workDirectory}/org/sonar/plugins/vbnet">
+                  <fileset
+                    dir="${analyzers.directory}/src/SonarAnalyzer.RuleDescriptorGenerator/bin/Release/net46/vbnet">
```

Comment:
[Educational question]: Why do we use the net46 version on line 258 and the netcoreapp3.1 version on line 267?

Reply:
Probably some historical reasons, as net46 existed before netcoreapp3.1 was added.

I don't really care, because I will remove the net46 build completely (for sure) and update the netcoreapp3.1 to net6 (hopefully)